### ISSUE TITLE
room and phonenumber in export

### ISSUE THREAD

**Issue by hoodie** [tdolist/tdo-rs#5]
```rust
if (user.has_phone() || user.has_room()) && TDO::export() {
    panic!("because room and number end up part of the name in the markdown file")
}
```

**Comment by h4llow3En** [comment]
This has to be fixed in [tdo-export](https://github.com/tdolist/tdo-export)

**Comment by h4llow3En** [comment]
fixed differently. I split the result at `,` and check if the first entry of the resulting vector ist `""`

**Comment by h4llow3En** [comment]
This will be included in the next version

**Comment by hoodie** [comment]
did you close or postpone this issue? 

>  @h4llow3En added this to the Version 2.0 milestone 4 hours ago
@h4llow3En h4llow3En closed this 4 hours ago

**Comment by Feliix42** [comment]
Afaik @h4llow3En fixed this in [`f81129d`](https://github.com/tdolist/tdo-export/commit/f81129d228ba8f92467e6e8c58e13e3d407c3f3c). In the upcoming release of `tdo` it will be fixed.

**Comment by h4llow3En** [comment]
Yes, this one is fixed with `tdo-export 0.2` as mentioned by @Feliix42
